geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/DataSerializerPropagationDUnitTest.java

```diff
@@ -375,53 +347,34 @@ public void testServerUpFirstClientLater() throws Exception {
     server1.invoke(() -> DataSerializerPropagationDUnitTest.registerDSObject1());
     server1.invoke(() -> DataSerializerPropagationDUnitTest.registerDSObject2());
 
-    server1.invoke(() -> DataSerializerPropagationDUnitTest.verifyDataSerializers(new Integer(2)));
+    server1.invoke(() -> DataSerializerPropagationDUnitTest.verifyDataSerializers(2));
 
-    client1.invoke(() -> DataSerializerPropagationDUnitTest
-        .createClientCache(NetworkUtils.getServerHostName(server1.getHost()), new Integer(PORT1)));
+    final String hostName = NetworkUtils.getServerHostName();
+    client1.invoke(() -> DataSerializerPropagationDUnitTest.createClientCache(hostName, PORT1));
 
-    client1.invoke(() -> DataSerializerPropagationDUnitTest.verifyDataSerializers(new Integer(2)));
+    client1.invoke(() -> DataSerializerPropagationDUnitTest.verifyDataSerializers(2));
 
     // Put some entries from the client
-    client1.invoke(new CacheSerializableRunnable("Put entries from client") {
-      @Override
-      public void run2() throws CacheException {
-        Region region = cache.getRegion(REGION_NAME);
-        for (int i = 1; i <= 10; i++) {
-          region.put(i, i);
-        }
+    client1.invoke(() -> {
```

Comment:
I think that everything in the test from this point on should be deleted.  It's not testing dataserializers at all - it's just exercising register-interest on integer keys and values.  It's a waste of time and removing it should get rid of the flaky-ness of this test.

Reply:
I think there's some value to asserting that the `DataSerializer` actually worked, by checking that the client can see values it put and that the server can see those values too. You're totally right about the part we wait for being useless, though. I'll take it out entirely.